content/docs/projects/reports-configurations/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Reports Configurations"
+description: ""
+lead: ""
+date: 2023-10-02T11:10:14+08:00
+lastmod: 2023-10-03T20:09:34+08:00
+draft: false
+images: []
+menu: 
+  docs:
+    parent: "others"
+weight: 999
+toc: true
+---
+
+## Reports Configurations
+### Overview
+ 
+ The "Reports Configurations" feature is a powerful addition to our system that empowers clients to tailor their report consumption experience to their unique business needs. Reports are essential tools for decision-making, but not all clients require the same data or presentation. With this feature, we offer a flexible solution that allows each client to customize report filters and columns according to their preferences.
+ Traditionally, Script Reports came with a one-size-fits-all approach. However, different clients may have distinct requirements. For instance, some may find certain filters or columns irrelevant to their business, while others might prefer to relabel columns to better align with their operations. This feature addresses these variances by enabling per-client configuration of reports.
+
+
+### Table of Contents
+ - [Why Was This Feature Added?](#why-was-this-feature-added)
+ - [Changes Made](#changes-made)
+ - [How to Use the Feature](#how-to-use-the-feature)
+ - [Testing](#testing)
+ - [Known Issues](#known-issues)
+ - [Future Improvements](#future-improvements)
+ - [References](#references)
+
+
+### Why Was This Feature Added?
+ 
+ This feature was introduced to enable per-client configuration of reports. Each client has unique preferences regarding how they consume reports. For example, some clients may wish to hide specific filters or columns that are irrelevant to their business needs, while others may want to relabel certain columns to provide better context for their business operations.
+
+### Changes Made
+#### On Load of the Report Document Form
+![On Load of the Report Document Form 1](image-6.png)
+ 
+  Adding implementation in `before_load` Form Event in report.js to fetch the filters and pass it to the backend.
```

Comment:
Let's decide on what voice and point of view to use in this documentation because in some parts we're using "we" and in others, we're using the passive voice.